wntr/epanet/io.py

```diff
@@ -1,13 +1,5 @@
 """
 The wntr.epanet.io module contains methods for reading/writing EPANET input and output files.
-
-.. rubric:: Contents
-
-.. autosummary::
-
-    InpFile
-    BinFile
-
```

Comment:
Should Content autosummary be removed throughout the code base?  I see some API docs where Contents is followed by Functions that I believe are created by the templates. For example, https://dbhart.github.io/WNTR/apidoc/wntr.metrics.economic.html

Reply:
I agree, we should do this across the board.

Reply:
Okay, this should be all fixed now.